gpib_server.py

```diff
@@ -190,9 +190,11 @@ def query(self, c, data):
         device will occur while the query is in progress.
         """
         instr = self.getDevice(c)
-        #instr.write(data)
-        #ans = instr.read()
-        ans = instr.query(data)
+        # instr.write(data)
+        # ans = instr.read()
+        # ans = instr.query(data)
```

Comment:
Maybe just remove the commented lines while you're in here? Git has history, so I say just nuke this stuff :)